src/app/shared/custom-components/toolbar/search/search.component.scss

```diff
@@ -1,10 +1,32 @@
-.example-form {
+@import '/src/app/styles/variables.scss';
+
+.search-form {
   min-width: 150px;
```

Comment:
I suggest smth/ like this:
.search-form {
  max-width: 260px; //we can remove this after changing tags component
  margin-left: 1em;

  &__field {
    display: flex;
    border: $border-primary;
    border-radius: 25px;
  }

  &__input {
    width: 100%;
    font-size: 20px;
    border: none;
    border-radius: 25px;
    outline: none;
  }

  &__button {
    color: $blue-primary; //extract this to button shared styles
  }